.packit.yaml

```diff
@@ -18,6 +22,9 @@ srpm_build_deps:
     - openssl-devel
 
 actions:
+    pre-sync:
+        - bash -c "./make-vendored-tarfile.sh ${PACKIT_PROJECT_VERSION}"
+        - bash -c "git restore Cargo.lock"
```

Comment:
Why do you need to restore the Cargo.lock file?

Reply:
the vendor script updates that, I'm just bringing it back to "normal"